### PR TITLE
add PPP, update UG/DG

### PR DESCRIPTION
**Pull request by Ben-Hanan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -221,7 +236,7 @@ Example:
 #### 3.3.4. Record leave taken by staff: `addl`
 
 Records leave taken by a staff that is in Eva. <br>
-Format: `addleave INDEX l/d/DATE [d/DATE]`
+Format: `addleave INDEX l/d/DATE [d/DATE] [l/d/DATE [d/DATE]]…​`
```
Comment (TheSpaceCuber) [inline docs/UserGuide.md]: is this addl instead?

Comment (Ben-Hanan) [reply docs/UserGuide.md]: right lol

Comment (Ben-Hanan) [reply docs/UserGuide.md]: i got a bit more commits to push anyways wait ah

File: docs/UserGuide.md [hunk 2]
```diff
@@ -244,13 +259,9 @@ Removes record of leave taken by staff. <br>
...
-* `list` followed by `deleteleave 2 d/09/09/2020` deletes the leave record of which the given date coincides with from the 2nd person in shown list.
-* `find -staff Betsy` followed by `deleteleave 1 d/09/09/2020` deletes the leave from the 1st person in the results of the `find` command.
-* `deleteleave 2 d/09/09/2020`
-
-#### 3.3.6. Edit leave taken by staff: `editl`
-[This feature is coming soon]
-
+* `list s-` followed by `dell 2 d/09/09/2020` deletes the leave record of which the given date coincides with from the 2nd person in shown list.
+* `find s- Betsy` followed by `dell 1 d/09/09/2020` deletes the leave from the 1st person in the results of the `find s-` command.
+* `delleave 2 d/09/09/2020`
```
Comment (TheSpaceCuber) [inline docs/UserGuide.md]: dell

Comment (Ben-Hanan) [reply docs/UserGuide.md]: oh

**Comment by TheSpaceCuber** [comment]
LGTM

**Pull request merged**